website/docs/cli/commands/test/index.mdx

```diff
@@ -44,8 +44,8 @@ the required conditions (assertions) are met. Once the test is complete, OpenTof
 
 Usage: `tofu test [options]`.
 
-This command will execute all `*.tftest.hcl` files in the current directory or in a directory called `tests`. You can
-customize this behavior using the [options](#options) below.
+This command will execute all `*.tftest.hcl`, `*.tftest.json`, `*.tofutest.hcl`, and `*.tofutest.json` files in the
```

Comment:
It's weird, but we never mentioned that test files can be in JSON format in our docs. So, I added it here.